paddle/operators/pool_op.cc

```diff
@@ -146,33 +142,30 @@ Pool3dOpMaker::Pool3dOpMaker(framework::OpProto *proto,
             "the number of channels, D, H and W is the depth, height and "
             "width of feature.");
 
-  AddAttr<std::string>("pooling_type",
-                       "PoolingType of pooling operator."
+  AddAttr<std::string>("poolingType",
+                       "(string), poolingType of pooling operator."
                        "Str constant equal to 'max' or 'avg'.")
       .InEnum({"max", "avg"});
-
   AddAttr<std::vector<int>>(
       "ksize",
-      "The pooling window size(depth, height, width) of pooling operator."
-      "If global_pooling = true, ksize is ignored and need not be "
+      "(vector ), the pooling window size(depth, height, width) of pooling "
```

Comment:
Pooling 3d and 2d seem to have very similar description documents, They can be merged into one `OpMaker` using the same description, like:

(vector), the pooling window size of pooling operator, for 3d pooling the layout should be (depth, height, width), for 2d it should be (height, width). ...